src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi4Driver.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace System.Device.Gpio.Drivers
+{
+    public unsafe partial class RaspberryPi4Driver : RaspberryPi3Driver
```

Comment:
should this inherit from GenericRaspberryPiDriver?

Reply:
Yes, of course. Fixed.